README.md

```diff
@@ -63,11 +63,11 @@ To build and test in Visual Studio, open
 Alternatively, you can build REEF.NET from a developer command line
 via:
 
-    msbuild .\lang\cs\Org.Apache.REEF.sln
+    msbuild .\lang\cs\Org.Apache.REEF.sln /p:Configuration="Debug" /p:Platform="x64"
 
 To test, execute the following command thereafter:
 
-    vstest.console.exe .\lang\cs\bin\x64\Debug\Org.Apache.REEF.Tests\Org.Apache.REEF.Tests.dll /Platform:x64
+    msbuild .\lang\cs\TestRunner.proj /p:Configuration="Debug" /p:Platform="x64"
```

Comment:
I tried that in first version of PR, didn't work. Likely because not all projects in solution have Test target, `/t:Test` failed complaining that Test target doesn't exist. I tried to work around this, but in the end a separate project turned out to be a lot easier to do.